apps/artemis-web/utils/hooks/use-graphql.d.ts

```diff
@@ -24,6 +24,7 @@ type optionsType = {
   isMutation?: boolean;
   running?: boolean;
   name?: string;
+  isTesting?: boolean;
```

Comment:
btw where is this actually defined? Is it inherited from an env var?

Reply:
YES! the default is false, I am using this flag for CYPRESS testing